model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -155,17 +164,30 @@ def __init__(self,
             fw_info: Framework information (e.g., mapping from layers to their attributes to quantize).
             return_float_outputs: Whether the model returns float tensors or not.
             wrapper: A function wrapper Pytorch Layers.
+            get_activation_quantizer_holder_fn: Function to retrieve a quantization holder for a node.
+
         """
         super(PytorchModel, self).__init__()
         self.graph = graph
         self.node_sort = list(topological_sort(graph))
-        self.nodes_dict = {}
+        self.node_to_activation_quantization_holder = {}
         self.append2output = append2output
         self.return_float_outputs = return_float_outputs
         self.fw_info = fw_info
         self.wrapper = wrapper
+        self.get_activation_quantizer_holder = get_activation_quantizer_holder_fn
         self._add_modules()
 
+    @property
+    def use_activation_holder_during_model_building(self) -> bool:
```

Comment:
Not very important but I think that this method can be located in the parent class "BaseModelBuilder" instead of being duplicated for Pytorch and Keras with the exact same implementation